indico/modules/events/templates/reviewing_questions_management.html

```diff
@@ -0,0 +1,105 @@
+{% from 'message_box.html' import message_box %}
+{% from '_sortable_list.html' import sortable_list %}
+
+<div class="manage-reviewing-questions">
+    <div class="toolbar space-after hide-if-locked">
+        <a class="i-button arrow icon-plus highlight" data-toggle="dropdown">
+            {%- trans %}Add new question{% endtrans -%}
+        </a>
+        <ul class="dropdown">
+            {% for name, field_type in field_types.iteritems() -%}
+                <li>
+                    <a class="js-action-button"
+                       data-href="{{ url_for(actions.create, event, field_type=name, **(args.create|default({}))) }}"
+                       data-title="{% trans %}Add custom reviewing question{% endtrans %}"
+                       data-ajax-dialog>
+                        {% trans name=field_type.friendly_name %}
```

Comment:
what's the point of this trans block?

![](https://i.imgur.com/FRflC.gif)

Reply:
Ah, indeed, the `friendly_name` is already translated here.

Reply:
Even if it was, this wouldn't help, since the string would not be extracted for translation (it might just work by coincidence if it was extracted somewhere else)